dev/integration_tests/web/web/.gitignore

```diff
@@ -0,0 +1,2 @@
+# This file is generated by a test. It should not be committed to source control.
+generated_entrypoint.html
```

Comment:
Why not git-ignore the entire directory `flutter_web_generated_entrypoint/`?

Reply:
`flutter_web_generated_entrypoint` is a temp directory outside the source tree that's created then deleted (see `Directory.systemTemp.createTempSync`). It's only used as a temporary space to dump an entire project. The test actually only needs the `index.html`. It doesn't need the "counter app" Dart code.